app/assets/stylesheets/govuk_publishing_components/components/_print-link.scss

```diff
@@ -20,6 +20,7 @@ $gem-c-print-link-background-height: 18px;
   background: image-url("govuk_publishing_components/icon-print.png") no-repeat govuk-spacing(2) 50%;
   background-size: $gem-c-print-link-background-width $gem-c-print-link-background-height;
   padding: govuk-spacing(2) govuk-spacing(2) govuk-spacing(2) (govuk-spacing(4) + $gem-c-print-link-background-width);
+  text-decoration: none;
```

Comment:
Wondering if the Button within Show Password should align (or the reserves)
![image](https://user-images.githubusercontent.com/71266765/120324033-0d80b880-c2de-11eb-8948-528a5d957af0.png)

Reply:
This was a choice by govuk accounts when they designed this pattern to add that little underline to try to differentiate it from our other button patterns (primary, secondary, other secondary etc).

As an answer to this and your other comments about buttons, I'd like to try and find a consistent way for us to do our custom secondary button pattern alongside our other button variants. The design system has a [specified secondary button pattern](https://design-system.service.gov.uk/components/button/#secondary-button) but govuk has it's own secondary button pattern mixed in, I suspect for legacy reasons. I don't think this is a problem I can solve in this PR or as part of this link style work.

From a logistics point of view it's marginally accessibility. Maybe I could make a card for our team and we could negotiate?

Reply:
> I don't think this is a problem I can solve in this PR

Agreed, sure I was thinking add an issue - add as a draft card for triage?